src/uct/ib/base/ib_iface.h

```diff
@@ -190,18 +190,21 @@ struct uct_ib_iface_config {
 enum {
     UCT_IB_CQ_IGNORE_OVERRUN         = UCS_BIT(0),
     UCT_IB_TM_SUPPORTED              = UCS_BIT(1),
+};
 
-    /* Indicates that TX cq len in uct_ib_iface_init_attr_t is specified per
-     * each IB path. Therefore IB interface constructor would need to multiply
-     * TX CQ len by the number of IB paths (when it is properly initialized). */
-    UCT_IB_TX_OPS_PER_PATH           = UCS_BIT(2)
+enum {
+    UCT_IB_CQ_CQE_ZIPPING            = UCS_BIT(0),
 };
 
 
 typedef struct uct_ib_iface_init_attr {
     unsigned    rx_priv_len;            /* Length of transport private data to reserve */
     unsigned    rx_hdr_len;             /* Length of transport network header */
-    unsigned    cq_len[UCT_IB_DIR_NUM]; /* CQ length */
+    struct {
+        size_t      cqe_size;            /* CQE size */
```

Comment:
maybe replace 'flags' and 'cqe_size' by 'cqe_zip_sizes' - bitmap of sizes for which cqe zip is supported and enabled